### PR TITLE
WITNESS ME

### PR DESCRIPTION
**Pull request by Cuboos**
My Wizard Sounds branch wouldn't let me commit any more files to it for some reason, i updated the changelog to include my sound additions.

### CODE REVIEW

File: html/changelog.html [hunk 1]
```diff
@@ -59,6 +59,8 @@ <h2 class="date">31 May 2015</h2>
 			<h3 class="author">duncathan updated:</h3>
 			<ul class="changes bgimages16">
 				<li class="tweak">Clarified the ability to retract changeling armblades.</li>
+                <li class="soundadd">Wizard Spells and Wands now have unique sounds, made by Cuboos</li>
```
Comment (Iamgoofball) [inline html/changelog.html]: This needs to be done with the proper changelog system

Comment (Cuboos) [inline html/changelog.html]: I fixed it, sorta, i did the right method but now how do i remove the HTML file?

File: html/changelogs/Cuboos-I'mImportantTooGais.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+author: Cuboos
+delete-after: True
+changes: 
+  -soundadd: "Wizard Spells and Wands/Staffs now have unique sounds"
+  -soundadd: Added Title3.ogg to the start up music
```
Comment (Jordie0608) [inline html/changelogs/Cuboos-I'mImportantTooGais.yml]: You're missing some quotation marks here

Comment (Cuboos) [inline html/changelogs/Cuboos-I'mImportantTooGais.yml]: Quotations added

**Comment by optimumtact** [comment]
Consider yourself witnessed, also you fucked up the changelog somehow

I'm gonna guess it's the ' in the changelog yaml file name.

**Comment by Aranclanos** [comment]
>more commits than lines changed
how

**Comment by Cuboos** [comment]
i... i don't know...

**Comment by Cheridan** [comment]
you tried

**Pull request closed without merging**